src/components/Facility/ConsultationDetails/QuestionnaireResponsesList.tsx

```diff
@@ -276,43 +278,59 @@ export default function QuestionnaireResponsesList({
 }: Props) {
   const { t } = useTranslation();
 
+  const { qParams, Pagination, resultsPerPage } = useFilters({
+    limit: 15,
+  });
+
+  const {
+    data: questionnarieResponses,
+    isFetching: questionnaireFetching,
+    isLoading: questionnaireLoading,
+  } = useQuery({
```

Comment:
since this is the primary request of this component, let's not alias it.

also do we really need both `isFetching` and `isLoading`?

```suggestion
  const {
    data: questionnarieResponses,
    isFetching: questionnaireFetching,
    isLoading: questionnaireLoading,
  } = useQuery({
```

Reply:
was doing extra checking of !fetching when no data was available to show
removed it now since we are checking for isLoading before data in ternary block